src/frontend/src/btc/components/send/BtcSendReview.svelte

```diff
@@ -18,4 +29,12 @@
 		}) || invalidAmount(amount);
 </script>
 
-<SendReview on:icBack on:icSend {source} {amount} {destination} disabled={invalid} />
+<SendReview on:icBack on:icSend {source} {amount} {destination} disabled={disableSend}>
+	<svelte:fragment slot="info">
+		{#if $hasPendingTransactionsStore === true}
+			<WarningBanner>{$i18n.send.info.pending_bitcoin_transaction}</WarningBanner>
```

Comment:
Can the warning appears after mount? If yes, I would add a `fade` or other effect to smoothen the appearance of the warning.

Reply:
Done!